src/plone/restapi/services/users/delete.py

```diff
@@ -5,6 +5,9 @@
 from zope.publisher.interfaces import IPublishTraverse
 
 
+FALSE_VALUES = (0, "0", False, "false", "no")
```

Comment:
@erral I am wondering if we should define this higher up to make sure it is used everywhere. This is not a blocker for a merge though.

Reply:
Yes, it could be useful indeed.